server/cmd/main.go

```diff
@@ -77,7 +77,10 @@ func main() {
 	viper.SetDefault("COMMITSHA", commitsha)
 	viper.SetDefault("RELEASE_CHANNEL", releasechannel)
 	viper.SetDefault("INSTANCE_ID", &instanceID)
-
+	viper.SetDefault("DBHOST", "localhost")
```

Comment:
```suggestion
	viper.SetDefault("DBHOST", "meshery-db")
```

Reply:
By default it should be localhost. It is overriden has meshery-db only in docker-compose. For k8s where it is in same pod, we want it to communicate over localhost where it by passes the networking stack and directly communicates via socket. Same for dev environment.